gui/src/components/JobRuns/List.js

```diff
@@ -2,58 +2,122 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import { withStyles } from '@material-ui/core/styles'
 import Table from '@material-ui/core/Table'
-import TableHead from '@material-ui/core/TableHead'
 import TableBody from '@material-ui/core/TableBody'
 import Typography from '@material-ui/core/Typography'
 import TableRow from '@material-ui/core/TableRow'
 import TableCell from '@material-ui/core/TableCell'
 import Card from '@material-ui/core/Card'
 import Link from 'components/Link'
 import TimeAgo from 'components/TimeAgo'
+import ReactStaticLinkComponent from '../ReactStaticLinkComponent'
+import classNames from 'classnames'
+import Button from 'components/Button'
 
-const styles = () => ({
-  jobRunsCard: {
-    overflow: 'auto'
+const styles = theme => {
+  return {
+    jobRunsCard: {
+      overflow: 'auto'
+    },
+    idCell: {
+      width: '40%'
+    },
+    stampCell: {
+      width: '40%'
+    },
+    statusCell: {
+      textAlign: 'end'
+    },
+    runDetails: {
+      paddingTop: theme.spacing.unit * 2,
+      paddingBottom: theme.spacing.unit * 2
+    },
+    status: {
+      paddingLeft: theme.spacing.unit * 1.5,
+      paddingRight: theme.spacing.unit * 1.5,
+      paddingTop: theme.spacing.unit / 2,
+      paddingBottom: theme.spacing.unit / 2,
+      borderRadius: theme.spacing.unit * 2,
+      width: 'fit-content',
+      display: 'inline-block'
+    },
+    failed: {
+      backgroundColor: '#e9faf2',
+      color: '#ff6587'
+    },
+    pending: {
+      backgroundColor: '#fef7e5',
+      color: '#fecb4c'
+    },
+    completed: {
+      backgroundColor: '#e9faf2',
+      color: '#4ed495'
+    }
   }
-})
+}
+
+// insight from - https://github.com/sindresorhus/titleize
+const titleize = input => {
```

Comment:
I feel like this should live in `gui/src/utils`. Can you also add a unit test for this?